public/css/theme.css

```diff
@@ -1,3 +1,5 @@
+@import url(http://fonts.googleapis.com/css?family=Lobster|Abel|Shadows+Into+Light|Pacifico|Orbitron);
```

Comment:
Also, why loading all those fonts? Especially because you're using `Abel` only :/